cmd/cloud-controller-manager/app/apis/config/v1alpha1/doc.go

```diff
@@ -20,11 +20,10 @@ limitations under the License.
 // call.
 
 // +k8s:deepcopy-gen=package
-// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/componentconfig
-// +k8s:conversion-gen=k8s.io/apimachinery/pkg/apis/config/v1alpha1
-// +k8s:conversion-gen=k8s.io/apiserver/pkg/apis/config/v1alpha1
+// +k8s:conversion-gen=k8s.io/kubernetes/cmd/cloud-controller-manager/app/apis/config
```

Comment:
kube-controller-manager is not needed? Also apiserver config and apimachinery config?

Reply:
kube-controller-manager types are linked below: `// +k8s:conversion-gen=k8s.io/kubernetes/pkg/controller/apis/config/v1alpha1`, which in turn registers the apiserver/apimachinery packages in the "package universe" of the conversion-gen program.

Reply:
:+1: